File: MODUL_1/PRAK110/PRAK110-2610817210027-Andre_Pratama.py
import math

print("Diketahui :")

Base = 5
Height = 12
print("Alas =", Base, "cm")
print("Tinggi =", Height, "cm")

SideA = Height
SideC = Base
SideB = math.sqrt(SideA**2 + SideC**2)

Length = SideA + SideB + SideC
Area = 0.5 * Base * Height

print("\nJawab : ")
print("Sisi A =", SideA, "cm")
print("Sisi B =", int(SideB), "cm")
print("Sisi C =", SideC, "cm")
print("Keliling =", int(Length), "cm")
print("Luas =", int(Area), "cm")

File: MODUL 1/PRAK110/PRAK110-2610817210027-Andre_Pratama.c
#include <stdio.h>
#include <math.h>

int main () {
    int base = 5;
    int height = 12;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", base);
    printf("Tinggi = %d cm\n", height);

    float sideA = height;
    float sideC = base;
    float sideB = sqrt(pow(sideA, 2) + pow(sideC, 2));
    
    float totalLength = sideA + sideB + sideC;
    float area = (base * height) / 2.0;

    printf("\nJawab : \n");
    printf("Sisi A = %d cm\n", (int)sideA);
    printf("Sisi B = %d cm\n", (int)sideB);
    printf("Sisi C = %d cm\n", (int)sideC);
    printf("Keliling = %d cm\n", (int)totalLength);
    printf("Luas = %d cm\n", (int)area);

    return 0;
}
